buidler/tasks/deploy/task.deploy.js

```diff
@@ -61,8 +61,7 @@ export default task(
       let deployer;
       if (taskArgs.signerindex && taskArgs.signer)
         throw Error("Can't have both signer and signer index");
-      if (!taskArgs.signerindex && !taskArgs.signer)
-        [deployer] = await ethers.getSigners();
+      if (!taskArgs.signerindex && !taskArgs.signer) deployer = getSysAdmin();
```

Comment:
Why do we want to do this? Atm it just makes everything more complicated. We should ideally only use 1 PK each. There is no benefit in using multiple PKs.

Reply:
see below, lets do this in the next refactor